sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -133,20 +133,26 @@ export class InstitutionController extends BaseController {
       );
     }
 
+    // Create User
+    const userEntity = this.userService.create();
```

Comment:
I believe that we should ensure that this user belongs to the institution that he is being added to.
I would recommend executing a bceidService.getAccountDetails to ensure the relationship on BCeID and we can also get the firstName, lastName from BCeID response.

Reply:
are we getting bceid user name?

Reply:
Resolved